internal/host/host.go

```diff
@@ -58,6 +58,8 @@ var WrongBootOrderIgnoreTimeoutStages = []models.HostStage{
 
 var InstallationTimeout = 20 * time.Minute
 
+var ReclaimTimeout = 60 * time.Minute
```

Comment:
My opinion is that reclaim timeout should be shorter.
Pod creation should be quite short.
Booting from discovery ISO is not dependent on getting ignition from MCS, so it should be also quite short.
Also, maybe consider making these timeouts separate.

Reply:
> Booting from discovery ISO is not dependent on getting ignition from MCS

No, but it does pull the rootfs, which is quite a bit larger.

Also the step to do the reclaim will pull the initrd and kernel from the service which are smaller, but still something we need to take into account.

Do you have any suggestions for times?

Reply:
It seemed to me that it needs to be shorter, but maybe you know better.
Anyway, they should be separate.  Pod creation should not take a long time (I think).

Reply:
I definitely don't know better.

I left this as long as it is because I have no idea how long these operations might take.
Given that we'll never succeed after we time out I felt that more time was better at least on the first pass.

If it turns out that this times out fairly frequently in a way that we can't do anything about, then maybe it will make sense to split or decrease the time, but for now I'd rather give it as much time as I can. What difference will timing out 10 or 20 minutes earlier really make?